src/test/kotlin/factory/PositionDevFactoryTest.kt

```diff
@@ -0,0 +1,18 @@
+package factory
+
+import enum.PositionType
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class PositionDevFactoryTest {
+
+  @Test
+  fun `Verify description created position DEV`(){
```

Comment:
Verify description created for position type DEV

Reply:
fix